packages/common/src/core/schemas/internal/follow/FollowCardSchema.ts

```diff
@@ -0,0 +1,44 @@
+import { IConfigurationSchema } from "../../types";
+
+export const FollowCardSchema: IConfigurationSchema = {
+  type: "object",
+  required: [],
+  properties: {
+    itemGalleryWithDefault: {
+      type: "array",
+      items: {
+        type: "string",
+      },
+      maxItems: 10,
+      default: [],
+    },
+    callToAction: {
+      type: "string",
+      default:
```

Comment:
hmm, I don't actually think we have a pattern in place for translating defaults in a schema - this may just need to be passed in from the frontend 🤔 I'll give it some thought